site/source/docs/porting/connecting_cpp_and_javascript/embind.rst

```diff
@@ -96,7 +96,14 @@ object.
    use whatever name you like for the module by assigning it to a new
    variable: ``var MyModuleName = Module;``.
 
+Binding libraries
+=================
+
+Binding code is run as a static constructor and static constructors only get run if the object file is included in the link, therefore when generating bindings for library files compiler must be explicitly instructed to include the object file.
```

Comment:
Can you wrap this line a 80 cols.   I know that other files here are not wrapped but I'm trying to move towards reasonable line lengths.